service/gcs/main.go

```diff
@@ -39,13 +39,22 @@ func memoryLogFormat(metrics *cgroupstats.Metrics) logrus.Fields {
 func readMemoryEvents(efdFile *os.File, cgName string, threshold int64, cg cgroups.Cgroup) {
 	// Buffer must be >= 8 bytes for eventfd reads
 	// http://man7.org/linux/man-pages/man2/eventfd.2.html
+	triggered := false
```

Comment:
I'm not convinced we should be blocking subsequent triggers of the event. Won't there be cases where memory usage could fluctuate, and we want to get the event each time? What is the problem this is trying to solve?

Reply:
Me and Howard were basically just trying to cut down on noise in the logs as it was hard to correlate to where this was actually impacting runc starting etc. I assumed we'd only really care about the first instance of this as the pods are generally short lived. If we want to log every time it goes over 50 I can remove it

Reply:
We don't know that pods are necessarily short lived. A pod could be kept running for days with different containers in it, for all we know. :)

Reply:
If knowing the first occurrence of the threshold being hit is helpful, let's just add a counter here and log that with the event info?

Reply:
No you're right, that was a dumb thought on my part. This was mostly to fix the short lived pod scenarios like functions etc where we're seeing an excess of these being fired for short lived pods but it'd be good to see for long lived scenarios the fluctuations.